src/common/config_opts.h

```diff
@@ -797,6 +797,7 @@ OPTION(osd_heartbeat_interval, OPT_INT, 6)       // (seconds) how often we ping
 OPTION(osd_heartbeat_grace, OPT_INT, 20)
 OPTION(osd_heartbeat_min_peers, OPT_INT, 10)     // minimum number of peers
 OPTION(osd_heartbeat_use_min_delay_socket, OPT_BOOL, false) // prio the heartbeat tcp socket and set dscp as CS6 on it if true
+OPTION(osd_heartbeat_min_size, OPT_INT, 2000) // the minimum size of OSD heartbeat messages to send
```

Comment:
No, we're using TCP/IP so we don't care at all. This is entirely so that our ping sizes are transmitted in jumbo-sized packets when our other messages are. 100% to catch the case where a network misonfiguration results in the server sending packets the switch won't accept, and so we transmit no replication data but don't get marked down (as our heartbeat pings are under the limit).